README.md

```diff
@@ -368,7 +368,7 @@ Parameters of the driver script, `abyss-pe`
  * `q`: minimum base quality [`3`]
  * `s`: minimum unitig size required for building contigs (bp) [`200`]
  * `S`: minimum contig size required for building scaffolds (bp) [`s`]
- * `t`: minimum tip size (bp) [`2k`]
+ * `t`: maximum length of dangling edges to trim [`k`]
```

Comment:
I don't know the correct terminology, but it should be consistent with the one of `ABYSS --help` (which I copied here and below). I will `s/dangling edges/blunt contigs/` here, in `Assembly/Options.cc` and `doc/abyss-pe.1`, ok?

Regarding the default value, I assume it means the if not specified, the value of `k` will be used: https://github.com/bcgsc/abyss/blob/master/Assembly/Options.cc#L343